src/components/CardScreenshot.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+
+function Card({ urlImage, image }) {
+  return (
+    <div>
+      <div className="w-72 h-72 bg-white border-2 border-gray-200 rounded-lg shadow dark:bg-gray-800 dark:border-gray-700 mb-8">
```

Comment:
Is it possible to move all the style into a separate style file? It is hard to read the code otherwise

Reply:
I will find out if it can be moved, or what can be done about it. If not I will organize the code better to make it more readable.